src/V3EmitC.cpp

```diff
@@ -1753,7 +1753,9 @@ class EmitCImp final : EmitCStmts {
     }
 
     void emitVarReset(AstVar* varp) {
-        AstNodeDType* dtypep = varp->dtypep()->skipRefp();
+        AstNodeDType* const dtypep = varp->dtypep()->skipRefp();
+        const string varName
```

Comment:
```suggestion
        const string varNameProtected
```

And likewise rename in other places.  This makes it easier to audit/understand what being printed is protected versus not.